### PR TITLE
[IMP] : business_requirement_project: related with Task 12797, add th…

### PR DESCRIPTION
**Pull request by martintorresvm**
…e related BR on Task creation
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/xie8899/project/pull/42%23discussion_r57678023%22%2C%20%22https%3A//github.com/xie8899/project/pull/42%23issuecomment-202742712%22%2C%20%22https%3A//github.com/xie8899/project/pull/42%23issuecomment-202749675%22%2C%20%22https%3A//github.com/xie8899/project/pull/42%23issuecomment-202750911%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/42%23issuecomment-202742712%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22cc%20%40elicoidal%20%22%2C%20%22created_at%22%3A%20%222016-03-29T06%3A48%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/16461806%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/victormartinelicocorp%22%7D%7D%2C%20%7B%22body%22%3A%20%22LGTM%22%2C%20%22created_at%22%3A%20%222016-03-29T07%3A11%3A38Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%2C%20%7B%22body%22%3A%20%22LGTM%22%2C%20%22created_at%22%3A%20%222016-03-29T07%3A16%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%207787f146218cf424dc4995b4f34939a26876ff69%20business_requirement_project/wizard/generate_projects.py%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/42%23discussion_r57678023%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22hum%3A%20this%20should%20normally%20not%20be%20here.%20Any%20reason%3F%22%2C%20%22created_at%22%3A%20%222016-03-29T06%3A41%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20business_requirement_project/wizard/generate_projects.py%3AL168-188%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 7787f146218cf424dc4995b4f34939a26876ff69 business_requirement_project/wizard/generate_projects.py 5'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/42#discussion_r57678023'>File: business_requirement_project/wizard/generate_projects.py:L168-188</a></b>
- <a href='https://github.com/elicoidal'></a> hum: this should normally not be here. Any reason?
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/42#issuecomment-202742712'>General Comment</a></b>
- <a href='https://github.com/victormartinelicocorp'></a> cc @elicoidal
- <a href='https://github.com/elicoidal'></a> LGTM
- <a href='https://github.com/duanyp1991'></a> LGTM

<a href='https://www.codereviewhub.com/xie8899/project/pull/42?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/xie8899/project/pull/42?approve=1'></a>&nbsp;<a href='https://github.com/xie8899/project/pull/42'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by xie8899** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/xie8899/project/pull/42'>Refresh</a>.<a href='#crh-update-none'></a>

File: business_requirement_project/wizard/generate_projects.py [hunk 1]
```diff
@@ -168,15 +168,20 @@ def _prepare_project_vals(self, br, parent):
 
     @api.multi
     def _prepare_project_task(self, line, project_id):
+        import pdb
+        pdb.set_trace()
```
Comment (elicoidal) [inline business_requirement_project/wizard/generate_projects.py]: hum: this should normally not be here. Any reason?

**Comment by martintorresvm** [comment]
cc @elicoidal

**Comment by elicoidal** [comment]
LGTM

**Comment by duanyp1991** [comment]
LGTM

**Pull request merged**